packages/material-ui/src/Modal/Modal.d.ts

```diff
@@ -20,7 +20,7 @@ export interface ModalProps
   keepMounted?: boolean;
   manager?: ModalManager;
   onBackdropClick?: React.ReactEventHandler<{}>;
-  onClose?: React.ReactEventHandler<{}>;
+  onClose?: (event: React.SyntheticEvent<any>, reason: string) => void;
```

Comment:
Please use `unknown` instead. It's the safer option compared to `any`. We want people to opt-in to loose typing rather than opt-in to strict typing.

```suggestion
  onClose?: (event: React.SyntheticEvent<unknown>, reason: string) => void;
```